import/repos.json

```
@@ -8,6 +8,10 @@
     "currentBranch": "master",
     "repo": "https://github.com/EventStore/EventStore",
     "branches": [
+      {
+        "version": "v21.10",
+        "name": "release/oss-v21.10"
```

```suggestion
        "name": "ylorph/doc_21.10"
```
need to be reset afterwerds to `release/oss-v21.10`